x-pack/heartbeat/monitors/browser/suite.go

```diff
@@ -66,6 +66,16 @@ func (s *Suite) FilterJourneys() synthexec.FilterJourneyConfig {
 	return s.suiteCfg.FilterJourneys
 }
 
+func (s *Suite) Fields() synthexec.StandardSuiteFields {
+	_, inline := s.InlineSource()
```

Comment:
could we `s/inline/isInline/` just reads more cleanly IMHO (I used to be a java programmer!)